src/stableSort.js

```diff
@@ -0,0 +1,41 @@
+var _curry2 = require('./internal/_curry2');
+
+/**
+ * Returns a copy of the list, sorted according to the comparator function, which should accept two values at a
+ * time and return a negative number if the first value is smaller, a positive number if it's larger, and zero
+ * if they are equal, values of the same key retain their original order. Please note that this is a **copy** of the list.
+ *  It does not modify the original.
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a,a -> Number) -> [a] -> [a]
+ * @param {Function} comparator A sorting function :: a -> b -> Int
```

Comment:
Let's remove `a -> b -> Int` here. It's confusing that it doesn't match `(a,a -> Number)` above, and at any rate it's unnecessary to duplicate the information.